File: Java/Maths/Sorting input numbers - from ascending to descending.java
import java.util.Scanner;
import java.util.Arrays;
class ascendingToDescending_input {
    public static void main(String args[]){
        Scanner input = new Scanner(System.in);
        System.out.print("How many numbers are there: ");
        int size = input.nextInt();
        int numbers[] = new int[size];
        for (int i = 0; i < numbers.length; i++) {
            System.out.print("Enter the number-"+(i+1)+": ");
            numbers[i] = input.nextInt();
        }
        int temp = 0;
        for (int i = 0; i < numbers.length; i++) {
            for (int x = 0; x < (numbers.length-1); x++) {
                if(numbers[x] > numbers[(x+1)]){
                    temp = numbers[x];
                    numbers[x] = numbers[(x+1)];
                    numbers[(x+1)] = temp;
                }
            }
        }
        System.out.println("\nNumbers are sorted from ASC to DSC: ");
        System.out.print(Arrays.toString(numbers));
    }
}

File: Python/Maths/Sorting input numbers - from ascending to descending.py
size = int(input("How many numbers are there: "))
numbers = [None] * size
for i in range(0, len(numbers)):
    numbers[i] = int(input("Enter the number-"+str(i+1)+": "))

temp = 0
for i in range(0, len(numbers)):
    for x in range(0, len(numbers)-1):
        if numbers[x] > numbers[(x+1)]:
            temp = numbers[x]
            numbers[x] = numbers[(x+1)]
            numbers[(x+1)] = temp

print("\nNumbers are sorted from ASC to DSC: ")
print(numbers)